embrace-android-delivery/src/main/kotlin/io/embrace/android/embracesdk/internal/delivery/scheduling/SchedulingService.kt

```diff
@@ -0,0 +1,20 @@
+package io.embrace.android.embracesdk.internal.delivery.scheduling
+
+import io.embrace.android.embracesdk.internal.capture.crash.CrashTeardownHandler
+import io.embrace.android.embracesdk.internal.delivery.intake.IntakeService
+
+/**
+ * This service is responsible for scheduling HTTP requests to the Embrace backend.
+ */
+interface SchedulingService : CrashTeardownHandler {
+
+    /**
+     * Start scheduling requests.
+     */
+    fun startScheduling()
```

Comment:
I don't think we need this to be external to the service. The logic to manage any checks for retries or network state change should be internal to the service, so `onPayloadIntake()` should be sufficient for external services to notify this bad boy to do its thing.